packages/core/src/Banner.js

```diff
@@ -2,48 +2,56 @@ import React from 'react'
 import Box from './Box'
 import Flex from './Flex'
 import Text from './Text'
-import Icon from './Icon'
 import CloseButton from './CloseButton'
-import Heading from './Heading'
 import PropTypes from 'prop-types'
 import styled, { withTheme } from 'styled-components'
-import { applyVariations, hasPaletteColor, deprecatedColorValue } from './utils'
+import {
+  Attention as AttentionIcon,
+  Information as InformationIcon,
+  Success as SuccessIcon,
+  Warning as WarningIcon
+} from 'pcln-icons'
```

Comment:
nb: This doesn't need to be changed but was wondering for the sake of backwards compatibility, is it still possible to import from `pcln-icons/lib`?

Reply:
I think that would destroy a lot of the tree shaking benefits

Reply:
I found the note in the migration docs.  See my comment there for further discussion.